src/orchestratord/src/controller/materialize.rs

```diff
@@ -516,6 +527,46 @@ impl k8s_controller::Context for Context {
             Ok(None)
         };
 
+        // console resources don't need to block on an explicit rollout, but
+        // we do want to wait to deploy the console until the environmentd is
+        // successfully up and running, or else it will crashloop trying to
+        // contact the service
```

Comment:
do we run into issues if we let it crash loop? offhand it seems like it should right it self once `envd` is up

Reply:
we're also doing a similar thing for the balancers because it was causing issues with the test suite which wants to check to make sure pods aren't crashing